client/blocks/app-banner/utils.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Internal dependencies
+ */
+import {
+	startsWith,
+	get,
+	reduce,
+} from 'lodash';
+
+export const EDITOR = 'editor';
+export const NOTES = 'notifications';
+export const READER = 'reader';
+export const STATS = 'stats';
+export const ALLOWED_PAGE_TYPES = [ EDITOR, NOTES, READER, STATS ];
+export const ONE_WEEK_IN_MILLISECONDS = 604800000;
+export const ONE_YEAR_IN_MILLISECONDS = 31540000000;
+
+export function getAppBannerData( translate, pageType ) {
+	switch ( pageType ) {
+		case EDITOR:
+			return {
+				title: translate( 'Rich mobile publishing.' ),
+				copy: translate( 'A streamlined editor with faster, simpler image uploading? Check and mate.' ),
```

Comment:
Oh I'm passing the translate from component as a parameter to this function. If I allowed child props I would have to move the functionality that renders the text (depending on section) to `layout/index.jsx`. Not sure if that's the cleanest way to do it? 

Actually, now I'm thinking about moving all of the helper methods from `utils.js` to component class methods, just to avoid passing props and translate around. Let me know what do you think about this.